server/text_generation_server/models/globals.py

```diff
@@ -52,6 +53,12 @@
 # index in all cases.
 ADAPTER_TO_INDEX: Optional[Dict[str, int]] = None
 
+PAGED_KV: bool
+if SYSTEM in {"rocm", "ipex"}:
+    PAGED_KV = False
+else:
+    PAGED_KV = True
+
```

Comment:
I think an approach like in #2575 might be better, it seems that ROCm support for flashinfer is in the works, so this should be determined in the device attention module.